QStreak.xcodeproj/project.pbxproj

```diff
@@ -203,6 +210,16 @@
 			path = QStreakUITests;
 			sourceTree = "<group>";
 		};
+		D5048AF8243F8FAE0029AE0C /* View Record */ = {
+			isa = PBXGroup;
+			children = (
+				D5048AF9243F90270029AE0C /* RecordDetailViewController.swift */,
+				D5048AFB243F90790029AE0C /* RecordDetailViewModel.swift */,
+				D5048AFD243F90950029AE0C /* RecordDetailViewController.storyboard */,
```

Comment:
NIT: Just so that we keep it consistent, order these as storyboard, controller, view model.